subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/settings/plugins/SubsSettingsPlugins.kt

```diff
@@ -44,3 +45,11 @@ class SubsSettingBuy @Inject constructor() : SubsSettingsPlugin {
         return SubsSettingBuyView(context)
     }
 }
+
+@ContributesMultibinding(scope = ActivityScope::class)
```

Comment:
It's weird that we're contributing the plugins in `ActivityScope` and the `BillingsModule` to `AppScope` 
Maybe `BillingsModule` should also be `ActivityScope`?

Reply:
I was literally discussing this with @cmonfortep yesterday and changed it in another PR but will do it here.

Reply:
+1